packages/nova-react/src/localization/nova-localization.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import type { NovaLocalization, FormatFn } from "@nova/types";
+import invariant from "invariant";
+
+const NovaLocalizationContext = React.createContext<NovaLocalization | null>(
+  null,
+);
+
+type NovaLocalizationProviderProps = {
+  localization: NovaLocalization;
+  children?: React.ReactNode | undefined;
```

Comment:
Nits: You can use the `React.PropsWithChildren` typing, but also do we even need to support `undefined` here with a provider?

Anyways, not in any way blocking.

Reply:
I'll fix that :D